com.unity.formats.fbx/Editor/FbxExporter.cs

```diff
@@ -1187,47 +1153,31 @@ internal static FbxVector4 ToFbxVector4(FbxDouble3 v)
             return new FbxVector4(v.X, v.Y, v.Z);
         }
 
-        internal static FbxDouble3 ConvertToRightHandedEuler(Vector3 rot)
-        {
-            rot.y *= -1;
-            rot.z *= -1;
-            return ToFbxDouble3(rot);
-        }
-
         /// <summary>
         /// Euler to quaternion without axis conversion.
         /// </summary>
         /// <returns>a quaternion.</returns>
         /// <param name="euler">Euler.</param>
-        internal static FbxQuaternion EulerToQuaternion(FbxVector4 euler)
+        internal static FbxQuaternion EulerToQuaternionZXY(Vector3 euler)
         {
-            FbxAMatrix m = new FbxAMatrix ();
-            m.SetR (euler);
-            return m.GetQ ();
+            var unityQuat = Quaternion.Euler(euler);
+            return new FbxQuaternion(unityQuat.x, unityQuat.y, unityQuat.z, unityQuat.w);
         }
 
-        /// <summary>
-        /// Quaternion to euler without axis conversion.
-        /// </summary>
-        /// <returns>a euler.</returns>
-        /// <param name="quat">Quaternion.</param>
-        internal static FbxVector4 QuaternionToEuler(FbxQuaternion quat)
+        internal static FbxQuaternion EulerToQuaternionXYZ(FbxVector4 euler)
```

Comment:
Update the comment:
Euler X/Y/Z rotation order to quaternion.
(and the return and param)